packages/trpc/server/middlewares/sessionMiddleware.ts

```diff
@@ -71,6 +71,7 @@ export async function getUserFromSession(ctx: TRPCContextInner, session: Maybe<S
       allowDynamicBooking: true,
       allowSEOIndexing: true,
       receiveMonthlyDigestEmail: true,
+      travelSchedules: true,
```

Comment:
Why are we adding `travelSchedules` to the user session object? I would want to keep this as learn as possible.

Reply:
removed it and added a separate getTravelSchedules handler instead